add-new-exercise.ps1

```diff
@@ -36,30 +36,34 @@ param (
 # Import shared functionality
 . ./shared.ps1
 
-$exerciseName = (Get-Culture).TextInfo.ToTitleCase($Exercise).Replace("-", "")
-$exercisesDir = Resolve-Path "exercises"
-$exerciseDir = Join-Path $exercisesDir $Exercise
-
 function Add-Project {
+    param (
+        [Parameter(Position = 0, Mandatory = $true)][string]$Exercise,
+        [Parameter(Position = 1, Mandatory = $true)][string]$ExerciseName
+    )
+
     Write-Output "Adding project"
 
-    $csProj = "$exerciseDir/$exerciseName.csproj"
+    $exercisesDir = Resolve-Path "exercises"
+    $exerciseDir = Join-Path $exercisesDir $Exercise
 
-    Run-Command "dotnet new xunit -lang ""C#"" --target-framework-override netcoreapp3.0 -o $exerciseDir -n $exerciseName"
-    Run-Command "dotnet sln ""$exercisesDir/Exercises.sln"" add $csProj"
+    $csProj = "$exerciseDir/$ExerciseName.csproj"
+
+    Invoke-CommandExecution "dotnet new xunit -lang ""C#"" --target-framework-override net5.0 -o $exerciseDir -n $ExerciseName"
```

Comment:
`Invoke-CommandExecution` sounds a bit odd to me. Is `Invoke-Command` also possible?

Reply:
`Invoke-Command` was my first try, but it's a reserved name.
Maybe `Invoke-ExpressionExitOnError` ?